### ISSUE TITLE
>>> from pandevice import firewall gives SyntaxError: invalid syntax

### ISSUE THREAD

**Issue by irom77** [PaloAltoNetworks/pandevice#12]
I got SyntaxError: invalid syntax for 'from pandevice import firewall'
```
$ python
Python 2.6.6 (r266:84292, May 22 2015, 08:34:51) 
[GCC 4.4.7 20120313 (Red Hat 4.4.7-15)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import pandevice

>>> from pandevice import firewall
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.6/site-packages/pandevice/firewall.py", line 32, in <module>
    from pandevice import device
  File "/usr/lib/python2.6/site-packages/pandevice/device.py", line 22, in <module>
    from base import PanObject, Root, MEMBER, ENTRY
  File "/usr/lib/python2.6/site-packages/pandevice/base.py", line 1095
    option_paths = {opt: re.sub(r"\([\w\d|-]*\)", opt, path) for opt in options}
                                                               ^
SyntaxError: invalid syntax
```

**Comment by btorresgil** [comment]
Hello.  Currently pandevice support python 2.7.x.  Looks like you are using python 2.6.6.  Pandevice may support python 3.x in the future, but there are no plans to support python 2.6.x at this time.

If you install python 2.7 and experience any similar issues, please let us know.

**Comment by irom77** [comment]
Oh right , it worked on 2.7

```
fw = firewall.Firewall('1.1.1.1', 'admin', 'password')
fw.op('show system info', xml=True)
'<response status="success"> ....'
```

But then I got error below when attempting to create address object ? 

```
>>> fw.add(objects.AddressObject("Webserver123", "123.123.123.123")).create()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/pandevice/base.py", line 487, in create
    pandevice.active().xapi.set(self.xpath_short(), element, retry_on_peer=self.HA_SYNC)
  File "/usr/local/lib/python2.7/dist-packages/pandevice/base.py", line 1562, in method
    raise the_exception
pandevice.errors.PanDeviceXapiError: Malformed Request
```

**Comment by btorresgil** [comment]
Glad to hear python 2.7 fixed it.  Pandevice currently support PAN-OS 7.0.x only.  Probably you are using an earlier version of PAN-OS, can you tell me what version?

We're working hard on other PAN-OS versions as a high priority.  The plan is to support all version of PAN-OS 6.0 and higher in a coming release.

**Comment by irom77** [comment]
Thank you . I have PA-5060s with PAN-OS 7.1.0

**Comment by btorresgil** [comment]
Actually, I take it back.  It's not because of a PAN-OS version issue.  This should work on PAN-OS 7.1 just fine.  The issue is that the second argument is the "type" of the address object, not the "value".  Here's the API doc for that object:

http://pandevice.readthedocs.io/en/latest/module-objects.html#pandevice.objects.AddressObject

I realize that most of my examples in the documentation are wrong, which is probably why you ran into an issue.  Rather than change the documentation, I'm going to change the API, because I think the way you're doing it is more clear and concise.

In the mean time, you can solve this issue by changing the command to specify the value:


    fw.add(objects.AddressObject("Webserver123", value="123.123.123.123")).create()

**Comment by irom77** [comment]
That's absolutely great ! Meantime I started to create objects;) Very cool, helpful when migrating to PaloAlto. I prefer to create new objects from pandevice instead of converting or importing my old, cluttered policy.

**Comment by btorresgil** [comment]
This is fixed as of version 0.3.5
